### ISSUE TITLE
Bareos-webui 20 installation failed (alpine)

### ISSUE THREAD

**Issue by barcus** [barcus/bareos#92]
```
➜  ~ docker run -t -i --rm alpine:edge
/ # apk update && apk add bareos-webui
fetch https://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/edge/community/x86_64/APKINDEX.tar.gz
v3.13.0-606-g81b6993b38 [https://dl-cdn.alpinelinux.org/alpine/edge/main]
v3.13.0-607-g413afbf74e [https://dl-cdn.alpinelinux.org/alpine/edge/community]
OK: 14012 distinct packages available
ERROR: unable to select packages:
  py3-selenium (no such package):
    required by: bareos-webui-20.0.0-r0[py3-selenium]
```

**Comment by barcus** [comment]
https://github.com/barcus/bareos/actions/runs/548911999

**Comment by barcus** [comment]
it seems to be fixed in 20.0.0-r2  🥳
